src/redux/reducers/experiments/experimentsDataDownloaded.js

```diff
@@ -0,0 +1,20 @@
+const experimentsDataDownload = (state, action) => {
```

Comment:
shouldn't this reducer use the new method that @cosa65 introduced to avoid the state unpacking?

Reply:
Whoops, these reducers should be removed